bot/constants.py

```diff
@@ -4,14 +4,14 @@
 
 from bot.bot import SeasonalBot
 
-__all__ = ('Client', 'Roles', 'bot')
+__all__ = ("Channels", "Client", "Roles", "bot")
```

Comment:
this should probably contain _all_ the classes, right?
```suggestion
__all__ = ("AdventOfCode", "Channels", "Client", "Colours", "Emojis", "Hacktoberfest", "Roles", "Tokens", "bot")
```

These are arranged alphabetically, and the actual classes should probably be moved around to match that ordering.

Reply:
No probs. Will do.